Mac client example (#123)
ufal/whisper_streaming: README.md

- nc is netcat with server's host and port

#### Client example on the Mac system:

> List microphones:

Reviewer: -> List of sounddevices

not only microphones are there

```
ffmpeg -hide_banner -f avfoundation -list_devices true -i ""
```

Live stream raw audio:
```

> ffmpeg -hide_banner -f avfoundation -i ":0" -ac 1 -ar 48000 -acodec pcm_s16le -ar 16000 -f s16le -loglevel error - | nc localhost 50000

Reviewer: Thanks.

`-ar 48000` seems to be redundant, because there is `-ar 16000`. 

And `-acodec pcm_s16le` and `-f s16le` seem to be duplicated.

Can you check if these options could be simpler? 

Also, can you expand what to do with list of microphones? You probably need to pass the `:0` from the list to the other command, right? Make it explicit.

- nc is netcat with server's host and port

#### Client example on the Mac system:

List of sounddevices:
```
ffmpeg -hide_banner -f avfoundation -list_devices true -i ""
```

> This command will produce a list of audio devices with their corresponding IDs. Use the ID in the next command with the -i option:

Reviewer 2: hey @SlavikCA maybe you can add something like this:

````suggestion
This command will produce a list of audio devices with their corresponding IDs. 
The output would look something like this:
```bash
...
[AVFoundation indev @ 0x123e05b20] AVFoundation audio devices:
[AVFoundation indev @ 0x123e05b20] [0] MacBook Air Microphone
[AVFoundation indev @ 0x123e05b20] [1] Device 1
[AVFoundation indev @ 0x123e05b20] [2] Device 2
...
```
Use the ID in the next command with the -i option. In this example I'm using a `MacBook Air` and the microphone is the device number `0`. 
````

- nc is netcat with server's host and port

#### Client example on the Mac system:

> List of sounddevices:

Reviewer 2: ```suggestion
List AVFoundation devices:
```

```
ffmpeg -hide_banner -f avfoundation -list_devices true -i ""
```

> This command will produce a list of audio devices with their corresponding IDs. 

Reviewer 2: ```suggestion
This command will produce a list of video and audio devices with their corresponding IDs. 
```
